File: 2. Loop Problems/Loop Problems in c/p25.c
// Write a C program to print Pascal triangle up to n rows

#include<stdio.h>

int main() {
    int n;
    printf("Enter N: ");
    scanf("%d", &n);

    for (int i = 0; i < n; i++) {
        int num = 1;

        for (int j = 0; j < n - i - 1; j++) {
            printf(" ");
        }

        for (int j = 0; j <= i; j++) {
            printf("%d ", num);
            num = num * (i - j) / (j + 1);
        }

        printf("\n");
    }

    return 0;
}

File: 2. Loop Problems/Loop problems in python/p25.py
# // Write a C program to print Pascal triangle up to n rows

n = int(input("Enter n: "))

for i in range (n):
    num = 1
    
    print(" "*(n-i-1),end="")
    
    for j in range(i+1):
        print(f"{num}",end=" ")
        num = num * (i-j) // (j+1)
        
    print() # moves to next line